### ISSUE TITLE
Building the training docker image fails

### ISSUE THREAD

**Issue by G-Thor** [cadia-lvl/unit-selection-festival#6]
This seems to be related to the EOL of python3.5, which is the latest python version for xenial. see here **https://stackoverflow.com/questions/65888892/why-i-cant-install-the-numpy-package-using-pip-on-ubuntu-16-04**

This could probably be solved by:

- fixing the numpy version to something compatible with python3.5
- installing python3.6< using e.g. deadsnakes ppa
- using ubuntu bionic or newer (if that doesn't break everything)

```
Step 16/21 : RUN pip3 install --upgrade pip 	&& pip3 install numpy 	&& pip3 install git+https://github.com/sequitur-g2p/sequitur-g2p@master
 ---> Running in 44715c82e76e
Collecting pip
  Downloading https://files.pythonhosted.org/packages/fe/ef/60d7ba03b5c442309ef42e7d69959f73aacccd0d86008362a681c4698e83/pip-21.0.1-py3-none-any.whl (1.5MB)
Installing collected packages: pip
  Found existing installation: pip 8.1.1
    Not uninstalling pip at /usr/lib/python3/dist-packages, outside environment /usr
Successfully installed pip-21.0.1
Traceback (most recent call last):
  File "/usr/bin/pip3", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/dist-packages/pip/__init__.py", line 16, in main
    from pip._internal.utils.entrypoints import _wrapper
  File "/usr/local/lib/python3.5/dist-packages/pip/_internal/utils/entrypoints.py", line 3, in <module>
    from pip._internal.cli.main import main
  File "/usr/local/lib/python3.5/dist-packages/pip/_internal/cli/main.py", line 60
    sys.stderr.write(f"ERROR: {exc}")
                                   ^
SyntaxError: invalid syntax
The command '/bin/sh -c pip3 install --upgrade pip 	&& pip3 install numpy 	&& pip3 install git+https://github.com/sequitur-g2p/sequitur-g2p@master' returned a non-zero code: 1
```

**Comment by judyfong** [comment]
Good to see it wasn't just me with this problem. i solved it by using the python3.7(?) docker image instead. But that image is like twice as big. maybe python3.7-slim would be enough.

**Comment by thdg** [comment]
You can fix this (at least) temporarily by doing `pip3 install --upgrade "pip < 21.0"`

**Comment by judyfong** [comment]
this is resolved now with the PR #5
